pyfr/integrators/dual/pseudo/pseudosteppers.py

```diff
@@ -203,6 +203,10 @@ def __init__(self, *args, **kwargs):
             'pyfr.integrators.dual.pseudo.kernels.rkvdh2pseudo'
         )
 
+    def dtau_mats_multiplied(self, y):
```

Comment:
This is going to be expensive.  Best resolved with a runtime scalar passed to the kernel telling it how to pre-scale the field.  Usually it can be 1.0 (so do nothing), but when we want to change the matrix we can set it, call the regular kernel.

Reply:
I could add the scalar factor in `pyfr/integrators/dual/pseudo/kernels/localerrest.mako#L15`: `vfac = alpha * min(${maxf}, max(${minf}, ${saff}*ufac));`.

Reply:
I added this change. We now modify the dtau field in the first pseudo-iteration, if the scaling multiplier alpha is not equal to 1.